src/util/properties.js

```diff
@@ -82,3 +86,34 @@ export default function gentleRegisterProperty (property, meta = {}, window = gl
 		}
 	}
 }
+
+/**
+ * Check if a CSS custom property can be registered.
```

Comment:
As someone looking at this package from the outside, it's unclear what this does. What does it mean "check if a custom property can be registered"? I would still call it `isRegistered()` and note the caveat that if it behaves identically to a non-registered property, this function will not detect that.